lib/src/presentation/bloc/genres_bloc.dart

```diff
@@ -0,0 +1,18 @@
+import '../../core/bloc/bloc_interface.dart';
+import '../../data/repository/genres_repository.dart';
+import '../../domain/usecase/genres_usecase_interface.dart';
+import '../../domain/usecase/implementation/fetch_genres_usecase.dart';
+
+class GenresBloc implements IBloc {
+  final GenresUseCase usecase = FetchGenresUsecase(repository: GenresRepository());
+
+  @override
+  void dispose() {}
+
+  void fetchGenres() => usecase.call();
```

Comment:
how you are communicating between the bloc and the use case? Because here you are calling the useCase but that data where is use it?